package.json

```diff
@@ -62,9 +62,9 @@
     "@types/sinon-chai": "^3.2.3",
     "@types/tmp": "^0.1.0",
     "balena-sync": "^10.0.0",
-    "blinking": "~0.0.2",
-    "bluebird": "^3.5.3",
-    "body-parser": "^1.12.0",
+    "blinking": "~0.0.3",
```

Comment:
This is not strictly updated, just setting the package.json version to match the package-lock.json